src/signatureUtils.js

```diff
@@ -16,7 +16,7 @@ const crypto = require('crypto')
 const loggerNamespace = '@adobe/aio-lib-events'
 const logger = require('@adobe/aio-lib-core-logging')(loggerNamespace,
   { level: process.env.LOG_LEVEL })
-const ADOBE_IOEVENTS_SECURITY_DOMAIN = 'https://adobeioevents.com'
+const ADOBE_IOEVENTS_SECURITY_SUBDOMAIN = 'https://static.adobeioevents.com'
```

Comment:
Would suggest to keep the variable name same here:
```suggestion
const ADOBE_IOEVENTS_SECURITY_DOMAIN = 'https://static.adobeioevents.com'
```